### DISCUSSION TITLE
How to integrate customized plots in dashboard?

### DISCUSSION THREAD

**Discussion by pmanandhar** [atoti/atoti#613] [Q&A] [answered]
Is it possible to integrate customized plots (such as seaborn or plotly or others) in the atoti dashboard? 

Thanks.

**Comment by CamilleBlanco** [comment, marked as answer]
Hi 👋 😄 

With the Community Edition of atoti, you cannot visualize graphs from other libraries in an atoti dashboard. Yet, you can interact with your graphs (add filters, etc.) with `session.visualize()`  or going to the session's homepage with `session.link()`. 

With [Atoti+](https://docs.atoti.io/0.6.6/atoti_plus.html), you can create application extensions (that you would precise in the `app_extensions` parameter when you create an [`atoti.Session()`](https://docs.atoti.io/0.6.6/how_tos/customization/customize_the_app.html#App-extensions) ), with which you could among other things add new types of widgets 📈
